### DISCUSSION TITLE
Refresh token & access token

### DISCUSSION THREAD

**Discussion by jhonnyminedr** [spencerwooo/onedrive-vercel-index#194] [Q&A] [answered]
I'm unable to get these, it's saying some error, is there any third party apps to get these (websites or script)
Thanks ❤️

**Comment by spencerwooo** [comment]
did you refer to the docs in the [previous project](https://github.com/spencerwooo/onedrive-cf-index#generating-onedrive-api-tokens) - which uses `beetcb/ms-graph-cli` to obtain the required tokens? i should believe they are still working, but if you are having problems please wait until i have some time to code a website for helping you get the tokens. hang tight, i should have some time next week.

**Reply by jhonnyminedr** [reply]
cannot get
in browser says success check panel
here it's
![Screenshot (142)](https://user-images.githubusercontent.com/95773407/147402612-7a66ce4e-d1c5-45d7-af97-2a81e40936cf.png)

**Comment by spencerwooo** [comment, marked as answer]
ok, new version released with a simplified deploy process, try it out ;)
